ToNote/ViewModels/MainViewModel.cs

```diff
@@ -99,6 +99,22 @@ public ICommand RemoveNoteCommand
                     var metadataFileName = note.Name + "Metadata.txt";
                     if (File.Exists(metadataFileName))
                         File.Delete(metadataFileName);
+
+                    var todosFileName = "*_" + note.Name + "_TODO";
+
+                    try
+                    {
+                        string[] todoFiles = Directory.GetFiles("Data", todosFileName);
+
+                        foreach (string file in todoFiles)
+                        {
+                            File.Delete(file);
+                        }
+                    }
+                    catch (FileNotFoundException)
```

Comment:
Empty catch block? God no. Use
`If (File.Exists(file) 
File.Delete(file);` , remove the try/catch blocks